Rubberduck.RegexAssistant/Pattern.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Rubberduck.RegexAssistant
+{
+    class Pattern
+    {
+        IRegularExpression RootExpression;
+        MatcherFlags Flags;
+
+        public Pattern(string expression, bool ignoreCase, bool global)
+        {
+            Flags = ignoreCase ? MatcherFlags.IgnoreCase : 0;
+            Flags = global ? Flags | MatcherFlags.Global : Flags;
```

Comment:
the flag initialization is a little irky. Currently they also serve no purpose whatsoever. That should change as soon as we actually get into functionality-stage